target/linux/ramips/dts/mt7621_sim_ax18t.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "sim,ax18t", "mediatek,mt7621-soc";
+	model = "SIM AX18T";
+
+	aliases {
+		label-mac-device = &gmac1;
+		led-boot = &led_status_red;
+		led-failsafe = &led_status_blue;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_blue;
+	};
+
+	chosen {
+		bootargs-override = "console=ttyS0,115200";
```

Comment:
> [    0.000000] Kernel command line: console=ttyS0,115200 firmware1 rootfstype=squashfs,jffs2

The u-boot kernel command contains an extra `firmware1`, and I prefer to override it. I'm not sure if I need to add the default `rootfstype=squashfs,jffs2` or `ubifs`? This does not affect kernel boot.
Reference: 51b61fd57059adeb10a3977e00772af2b76dc40b

Reply:
it make no difference, the current kernel never care about the 'firmware1' args in boot-args.

Reply:
> [    0.000000] Unknown kernel command line parameters "firmware1", will be passed to user space.

Thank you for your reply. Yes, you are right, but it can remove this confusing log. Anyway, that's not a very important change.